webhook/aws/json_helper.go

```diff
@@ -0,0 +1,57 @@
+package aws
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"fmt"
+)
+
+const (
+	MEDIATYPE_JSON = "application/json"
+)
+
+var (
+	ErrJsonEmpty   = errors.New("JSON payload is empty")
+)
+
+func DecodeJsonPayload(req *http.Request, v interface{}) ([]byte, error) {
+	if req.Body == nil {
+		return nil, ErrJsonEmpty
+	}
+
+	payload, err := ioutil.ReadAll(req.Body)
+	req.Body.Close()
+	if err != nil {
+		return nil, err
+	}
+	if len(payload) == 0 {
+		return nil, ErrJsonEmpty
+	}
+	err = json.Unmarshal([]byte(payload), v)
```

Comment:
As written, any numbers will be returned as float64 vs a number.  You may want to use func UseNumber() if you need an accurate integer.